toxcore/group.c

```diff
@@ -147,12 +181,12 @@ static Group_c *get_group_c(const Group_Chats *g_c, int groupnumber)
  * TODO(irungentoo): make this more efficient.
  */
 
-static int peer_in_chat(const Group_c *chat, const uint8_t *real_pk)
+static aint peer_in_chat(const Group_c *chat, const uint8_t *real_pk)
 {
-    uint32_t i;
+    size_t i;
```

Comment:
Why did this change to `size_t`? It's not being compared to a `size_t` anywhere.